### ISSUE TITLE
md5 not working properly

### ISSUE THREAD

**Issue by shubham735** [shubham735/crackhash_ss#1]

**Comment by swoutch** [comment]
What is your test case?

When I test `crackhashss.py` with the 1st line of `md5.txt` (`f447b20a7fcbf53a5d5be013ea0b15af`), it says the password is `12345678`.

However when I hash the string `12345678`, it is not in the `md5.txt` file :
```python
import hashlib

print(hashlib.md5(b"12345678").hexdigest())
# 25d55ad283aa400af464c76d713c07ad
```
```bash
grep 25d55ad283aa400af464c76d713c07ad data/md5.txt
```

I tried adding a line break at the end of the string, and the 3rd line of `md5.txt` matches (but not the 1st as expected) :
```python
print(hashlib.md5(b"12345678\n").hexdigest())
# 23cdc18507b52418db7740cbb5543e54
```
```bash
grep 23cdc18507b52418db7740cbb5543e54 data/md5.txt
```

So I think there are 2 issues here:
- the file `md5.txt` contains the hash of the passwords + a line break (which I think is not the goal)
- the file `md5.txt` seems incorrect/shifted in comparison with `tenmillion.txt`

I think the first thing to do is writing a little script that take `tenmillion.txt` in input and generate the (correct) `md5.txt` file.

**Comment by shubham735** [comment]
I am working on it along with another project, In mean time if you come up with better solution Please make a pull request.

**Comment by shubham735** [comment]
Issue resolved v0.1.1
